coreneuron/mpi/nrnmpi.cpp

```diff
@@ -30,6 +30,7 @@ THE POSSIBILITY OF SUCH DAMAGE.
 #include <sys/time.h>
 
 #include "coreneuron/nrnconf.h"
+#include "coreneuron/apps/corenrn_parameters.hpp"
```

Comment:
I think that this is not needed anymore?

Reply:
yes! thanks Ioannis!

Reply:
of course, sorry